Examples/Tests/pec/inputs_test_2d_pec_field_insulator

```diff
@@ -15,7 +15,7 @@ geometry.prob_hi = 1.e-2  3.e-2
 
 # Boundary condition
 boundary.field_lo = neumann      periodic
-boundary.field_hi = PECInsulator periodic
+boundary.field_hi = pec_insulator periodic
```

Comment:
This is an incidental fix, making the name consistent with what is in the documentation.